kombu/transport/SQS.py

```diff
@@ -377,8 +384,8 @@ def basic_ack(self, delivery_tag, multiple=False):
         except KeyError:
             pass
         else:
-            self.asynsqs.delete_message(message['sqs_queue'],
-                                        sqs_message['ReceiptHandle'])
+            self.sqs.delete_message(QueueUrl=message['sqs_queue'],
```

Comment:
So deleting messages isn't async anymore?
What's the cause of this change?

Reply:
So deleting messages isn't async anymore?
> Yes. 
What's the cause of this change?
> Due async delete, Worker was not able to delete any message completed after warm shutdown. 
Eg: We have a message:M1 and worker1 is process that message. When worker Warm Shutdown, MainProcess is waiting till M1 processing done but due to async delete system exit before deleting that message from SQS.